doc/sle15_features_in_partitioner.md

```diff
@@ -63,91 +63,148 @@ with some rough ideas.
 
-BCache technology allows to create software hybrid disks. The idea is to use big rotational disks as persistent storage (backing devices) and then to use a fast (e.g., SSD) device as cache (caching devices) for it. Backing and caching devices could be whole disks or even partitions.
+## Features 2 and 3: Format/mount/encrypt whole disks, Partitionable Software RAIDs
 
-### Research
+Right now the Expert Partitioner allows to format/mount/encrypt Software RAIDs but they cannot be partitioned. And for Disks is just the opposite situation. Disks can be partitioned but they cannot be formatted/mounted/encrypted.
 
-How it works? BCache has backing and caching devices. It is grouped in sets
-( multiple backing and single cache but maybe in future multiple caches will
-be possible ) and this set is then registered to bcache device like bcache0
-which can be formatted and mounted.
+This feature is about adding what is missing in each case. So the Expert Partitioner should allow to format/mount/encrypt whole disks and also it should allow to add partitions to Software RAIDs. All these features are already supported by libstorage-ng.
 
-At the end of [this document](https://bcache.evilpiepirate.org/) there is an example. It seems that there is a `/dev/bcacheX` device for each backing device (no sets of backing devices). And a caching device can be use as cache for several backing devices.
+At the end, Software RAIDs and Disk devices should be almost identical. The only one difference
+is that Software RAIDs can be created by the user.
 
-For example:
+### Selected solution
 
-```
-make-bcache -B /dev/sdc /dev/sdd -C /dev/sda2
-```
+The idea is to keep the traditional layout of the partitioner while being more
+precise when offering buttons to act on the devices. Displaying all the buttons
+that apply to a given device while hiding any button that makes no sense, even
+if that means dinamically changing the set of buttons everytime a new element
+is selected in a table. In addition, generic labels that may lead to different
+screens (like "edit" and "resize") should be avoided. Different actions will
+always be identified by different labels.
 
-* creates `/dev/bcache0` for backing device `/dev/sdd`
-* creates `/dev/bcache1` for backing device `/dev/sdc`
-* attaches `/dev/sda2` as cache for `/dev/sdd`
-* attaches `/dev/sda2` as cache for `/dev/sdc`
+#### "Hard Disks" view 
 
-Also, it seems that the caching device can be a set of devices, but according to documentation this feature is not supported yet: "Cache devices are managed as sets; multiple caches per set isn't supported yet but will allow for mirroring of metadata and dirty data in the future."
+When a disk is selected in the list, the following buttons will be displayed.
 
-### Open questions
+  * "Edit": to open the wizard to format/mount/encrypt the device (instead of
+    the historical behavior of opening the "Partitions" tab for the disk).
+  * "Add Partition"
```

Comment:
So no "delete all partitions" or "erase"?

Reply:
If we add "Delete Partitions" here, we also need to add it to the RAIDs (for symmetry). And there we already have "Add RAID", "Edit", "Add Partition" and "Delete". It wouldn't fit.

Reply:
So buttons like in past for "Add..." or "Delete..." with submenu is not planned, right?

Reply:
I wouldn't add them on this first iteration. Maybe we will end up there, but let's try to avoid it in a first round.

Reply:
ok